Tests/RxCocoaTests/DelegateProxyTest+WebKit.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  DelegateProxyTest+WebKit.swift
+//  Tests
+//
+//  Created by Giuseppe Lanza on 14/02/2020.
+//  Copyright © 2020 Krunoslav Zaher. All rights reserved.
+//
+
+#if os(iOS) || os(macOS)
```

Comment:
I think you don't need this entire file, the delegate proxy is already tested on UIKit level. (For example there isn't a specific `DelegateProxyTest+UITableView`). 

As long as you have a `WKWebViewTests`, that should suffice:]

Reply:
WKWebView behaves differently in dealloc. Dealloc doesn't happen immediately as expected so I preferred to test as it was tested for UIWebView before.

Reply:
Right, but you are testing a lot of non-dealloc behavior (e.g. copying the "doThisTest" stuff etc). Maybe worth just testing the specific deallocation? Or is there any reason to test the surrounding?

Reply:
Code coverage mainly. It's a new file, with a new Delegate proxy. Without this test `RxWKNavigationDelegateProxy` won't be covered and while we can agree that it follows the same pattern as others we can't be sure about future refactoring and human errors.

Reply:
Alright. I don't mind much about coverage in this case (not everything has to be covered), but I don't mind the extra test.